app/models/concerns/smooch_language.rb

```diff
@@ -4,9 +4,10 @@ module SmoochLanguage
   extend ActiveSupport::Concern
 
   module ClassMethods
-    def get_user_language(uid, message = {}, state = nil)
-      team = Team.find(self.config['team_id'])
-      default_language = team.default_language
+    def get_user_language(uid, message = {}, state = nil, team_id = nil)
+      team_id ||= self.config['team_id']
+      team = Team.find_by_id(team_id)
```

Comment:
Why we need receive `team_id` as a parameter? I thought that in this context we should always have a `team_id` in the config.

Reply:
Removed it and depend on `team_id` in the config